test/unit/models/hospitalAdministrator.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+const assert = require('assert');
+const helper = require('../../helper');
+const models = require('../../../models');
+
+describe('models.HospitalAdministrator', function () {
+  beforeEach(async function() {
+    await helper.loadFixtures(['hospital']);
```

Comment:
Here you can see how I use the helper to load fixture data into the db before the following test. This is so that I have a valid Hospital record before testing creating a HospitalAdministrator.